libnavui-dropin/src/main/res/layout/mapbox_scalebar_placeholder_layout.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    tools:parentTag="android.widget.FrameLayout">
+
+    <View
+        android:id="@+id/scalebarPlaceholder"
+        android:layout_width="match_parent"
```

Comment:
Should this be `wrap_content`? @dzinad

Reply:
I don't think we need to complicate this anymore as it already is given the limitations. Since we don't control the width of `scalebar` and display it to be of the size as `MapView` specifies, I don't think there is any need to make the `FrameLayout` and `this view` as `match_parent`. Because when the scalebar becomes visible, even though it takes up half the screen, the `FrameLayout` takes the entire width. In my opinion we should make both the scalebar `FrameLayout` and the above view to be `wrap_content`